### PR TITLE
Add password reset service

### PR DESCRIPTION
**Pull request by markiewiczm**

### CODE REVIEW

**Review by MercinQ — APPROVED** [review]

File: Docs/README_publish.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	        proxy_set_header Upgrade $http_upgrade;
+	        proxy_set_header Connection keep-alive;
+	        proxy_set_header Host $host;
+	        proxy_cache_bypass $http_upgrade;
+    		}
+	}
+7. Run nginx: 
+	-sudo nginx-t
+	-sudo service nginx start
+8. Execute command: dotnet publish na Windowsie
```
Comment (Grosbo) [inline Docs/README_publish.txt]: "dotnet publish na Windowsie"

File: ez-bet-app/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see https://editorconfig.org
```
Comment (Grosbo) [inline ez-bet-app/.editorconfig:1]: add to .gitignore

File: Docs/README_publish.txt (hunk 1 above)
Comment (Grosbo) [inline Docs/README_publish.txt]: "dotnet publish na Windowsie"

File: ez-bet-app/.gitignore [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```
Comment (Grosbo) [inline ez-bet-app/.gitignore:1]: add mentioned things

File: ez-bet-app/.vscode/launch.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (Grosbo) [inline ez-bet-app/.vscode/launch.json:1]: add to .gitignore

File: ez-bet-app/src/app/app.component.html [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<div>
+  <h1>
+    Welcome
+  </h1>
+  <app-game-list></app-game-list>
```
Comment (Grosbo) [inline ez-bet-app/src/app/app.component.html:5]: Remove nested componenet.

**Review by Grosbo — APPROVED** [review]

**Pull request closed without merging**